lib/fog/vsphere/models/compute/tickets.rb

```diff
@@ -0,0 +1,29 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Tickets < Fog::Collection
+        autoload :Ticket, File.expand_path('../ticket', __FILE__)
+
+        model Fog::Compute::Vsphere::Ticket
+
+        attr_accessor :server
+
+        def create(opts = {})
+          requires :server
+          case server
+            when Fog::Compute::Vsphere::Server
+              new(
+                service.vm_acquire_ticket(
+                  opts.merge(
+                    'instance_uuid' => server.instance_uuid
+                  )
+                )
+              )
+            else
+            raise 'server is required to create a ticket'
+          end
```

Comment:
Maybe you should just raise an exception if is not a `Fog::Compute::Vsphere::Server` to be more explicit?